docker-compose.yml

```diff
@@ -1,5 +1,16 @@
 version: "3.3"
 services:
+  postgres:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB}
+    ports:
+      - 5432:5432
```

Comment:
If seem to remember having issues with mapping to the port 5432 on my local machine, as it was the port my local postgres installation was using and it generated a conflict. Just `- 5432` will map to an available port.

Reply:
That is indeed an issue if you already have a postgres installed on your machine. If we map it to an available port that's not `5432`, the user will still have issues running the services because of the postgres connection strings `DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@host.docker.internal:5432/${POSTGRES_DB}` .
Another option is to recommend not to run both instances at the same time

Reply:
or perhaps just choose a different port altogether—say 5433 for example. IAccording to [this](https://stackoverflow.com/questions/55115080/how-to-specify-different-port-for-a-docker-postgres-instance), then we would need to change the mapping to:
```yml
ports:
      - 5433:5432
```
And the `DATABASE_URL` to:
```
DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@postgres:5432/${POSTGRES_DB}
```

I haven't tested this, so I'm not sure if it would work, but it could be a good solution.

PS: If we go this route perhaps we can rename the docker compose service from `postgres` to `local.db` as in the example, to make it less confusing.

Reply:
I like this actually, great idea, let me test if everything works and then I'll commit the change

Reply:
Seems to be working with the settings as in [e5c9a28](https://github.com/diogomatoschaves/MyCryptoBot/pull/134/commits/e5c9a28d60c393ff0f9f4e22d35d3af9697fb3d9) , just changing the host port and the port in the connection strings. I also had to modify django's `settings.py` to connect to port `5433` . Please double check if everything works correctly on your end too.